index.bs

```diff
@@ -2367,43 +2367,29 @@ To <dfn>get the browsing context info</dfn> given |context|,
 
    Note: This includes the fragment component of the URL.
 
-1. Let |child info| be the result of [=get the descendent browsing contexts=]
-   given |context id|, |depth| + 1, and |max depth|.
+1. Let |child infos| be a list.
```

Comment:
This is wrong; if `|max depth|` is 0 we want to return `null` (or maybe only if there are actually child contexts?) The point is we only want to return the empty list if there actually aren't any children; we use `null` to signify that we reached the maximum dpeth.

Reply:
Oh yes! Good catch. In my implementation I actually use `null` but forgot to change it here. Will be fixed with the next update.